lib/aca_entities/error.rb

```diff
@@ -7,7 +7,7 @@ module Error
     module ErrorInitalizer
       attr_reader :original
 
-      def initialize(msg, original = $!)
+      def initialize(msg, original = $ERROR_INFO)
```

Comment:
pulled this code from Avdi guidance.  it appears $ERROR_INFO is functionally equivalent to $! but in context see it requires:

required 'English'